client/src/components/Layout/EventInfo/EventInfo.js

```diff
@@ -1,8 +1,77 @@
 import React, { Component } from 'react'
 import './eventInfo.css'
 import Background from '../../../Images/background.jpg'
+import ClockIcon from '../../../Images/icons8-clock-128.png'
+import CalendarIcon from '../../../Images/icons8-calendar-64.png'
+import LocationIcon from '../../../Images/icons8-location-96.png'
+import NoteIcon from '../../../Images/icons8-note-48.png'
+import PersonIcon from '../../../Images/icons8-person-60.png'
 
+const gapi = window.gapi
+const CLIENT_ID = '558072145685-hl8laqhvbqcd0f5vfrnt61v1ts1ls5lh.apps.googleusercontent.com'
+const API_KEY = 'AIzaSyBaXfE46xxRXpcw7vSUmIZaKlwJrec1bGY'
+const DISCOVERY_DOCS = ['https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest']
+const SCOPES = 'https://www.googleapis.com/auth/calendar'
```

Comment:
This should come from a [config file](https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5).

Reply:
Yes, I will work on it , its just a test